core/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/split/SplitOpDesc.scala

```diff
@@ -2,23 +2,42 @@ package edu.uci.ics.amber.operator.split
 
 import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
 import com.google.common.base.Preconditions
+import com.kjetland.jackson.jsonSchema.annotations.{
+  JsonSchemaInject,
+  JsonSchemaString,
+  JsonSchemaTitle
+}
 import edu.uci.ics.amber.core.executor.OpExecWithClassName
 import edu.uci.ics.amber.core.virtualidentity.{ExecutionIdentity, WorkflowIdentity}
 import edu.uci.ics.amber.core.workflow._
 import edu.uci.ics.amber.operator.LogicalOp
+import edu.uci.ics.amber.operator.metadata.annotations.HideAnnotation
 import edu.uci.ics.amber.operator.metadata.{OperatorGroupConstants, OperatorInfo}
 import edu.uci.ics.amber.util.JSONUtils.objectMapper
 
-import scala.util.Random
 class SplitOpDesc extends LogicalOp {
 
-  @JsonProperty(value = "split percentage", required = false, defaultValue = "80")
-  @JsonPropertyDescription("percentage of data going to the upper port (default 80%)")
+  @JsonSchemaTitle("Split Percentage")
+  @JsonProperty(defaultValue = "80")
+  @JsonPropertyDescription("percentage of data going to the upper port")
   var k: Int = 80
 
-  @JsonProperty(value = "random seed", required = false)
-  @JsonPropertyDescription("Random seed for split")
-  var seed: Int = Random.nextInt()
+  @JsonSchemaTitle("Random Shuffle")
```

Comment:
I suggest  change this to “Auto-generate seed” to avoid confusion.

Reply:
Renamed.